libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -203,7 +204,7 @@ export class CheckboxComponent implements ControlValueAccessor, OnInit, OnDestro
         this.muteKey(event);
     }
 
-    /** @hidden Updates checkbox Indeterminate state on spacebar key on IE11 */
+    /** @hidden Updates checkbox Indeterminate state on space bar key on IE11 */
```

Comment:
Actually both are possible: https://en.wikipedia.org/wiki/Space_bar :D You can leave it with the space. Will add it to the `Cozy vs Cosy` discussion :D